src/mui/layout/Sidebar.js

```diff
@@ -6,29 +6,43 @@ import Paper from 'material-ui/Paper';
 import Responsive from './Responsive';
 import { setSidebarVisibility as setSidebarVisibilityAction } from '../../actions';
 
-const styles = {
+
+export function getStyles(props, context) {
+  const {
+    sideBar,
+  } = context.muiTheme;  
+  const styles = {
     sidebarOpen: {
-        flex: '0 0 16em',
+        flex: !(sideBar && sideBar.width) ? '0 0 16em' : `0 0 ${sideBar.width}`,
```

Comment:
please don't negate the condition, it makes it harder to reason about. Same for the others.